src/test/java/uk/gov/hmcts/reform/rsecheck/PojoTest.java

```diff
@@ -39,14 +40,16 @@
         GetTaskDmnRequest.class,
         GetTaskDmnResult.class,
         SendMessageRequest.class,
-        ProcessVariables.class
+        ProcessVariables.class,
+        TaskToCreate.class
     };
     // Cannot test equals for generic classes
```

Comment:
suggestion: Lombok generated methods are tested code so you used lombok you could safely skip them for unit testing since it's already covered.